js/game.js

```diff
@@ -9,8 +11,80 @@ function whack(rat) {
   rat.classList.add('hidden')
 }
 
+/**
+ * Hide all the rats.
+ *
+ * @param {Nodelist} rats All the rats.
+ */
+function hideRats(rats) {
+    rats.forEach(function(rat) {
+        rat.classList.add('hidden')
+    })
+}
+
 rats.forEach(function(rat) {
```

Comment:
this is being run on page load, but doesnt need to run until the game starts. can we move this into a function that is called when the game starts